apps/api/src/server.ts

```diff
@@ -20,16 +20,44 @@
 import { demoApplication } from '@pcgl-daco/data-model/src/main.js';
 import cors from 'cors';
 import express, { Request, Response } from 'express';
+import path, { dirname } from 'path';
+import * as swaggerUi from 'swagger-ui-express';
+import { fileURLToPath } from 'url';
+import yaml from 'yamljs';
+
+import { getHealth, Status } from './app-health.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 
 export const port = process.env.PORT || 3000;
 
 const startServer = async () => {
 	const app = express();
 
+	app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(yaml.load(path.join(__dirname, './resources/swagger.yaml'))));
+
 	app.get('/', (_req: Request, res: Response) => {
 		res.send('Hello World!');
 	});
 
+	app.get('/health', (_req: Request, res: Response) => {
+		const health = getHealth();
+
+		// TODO: implement proper versioning logic when in place
```

Comment:
Good suggestion, could leverage `const { npm_package_version } = process.env;` then replace V1 in resbody.version